core/src/main/java/org/frankframework/jdbc/dbms/TransactionalDbmsSupportAwareDataSourceProxy.java

```diff
@@ -85,35 +87,56 @@ public String getDestinationName() throws SQLException {
 	@Override
 	public Connection getConnection() throws SQLException {
 		Connection conn = super.getConnection();
-		if(metadata == null) {
+		if (metadata == null) {
 			log.debug("populating metadata from getConnection");
 			populateMetadata(conn);
 		}
-
 		return conn;
 	}
 
 	@Override
 	public String toString() {
-		if(metadata != null && log.isInfoEnabled()) {
+		if (metadata != null && log.isInfoEnabled()) {
 			return getInfo();
 		}
 		return obtainTargetDataSource().toString();
 	}
 
 	public String getInfo() {
-		StringBuilder builder = new StringBuilder();
-
-		if(metadata != null) {
-			builder.append("user ["+metadata.get("user")+"]");
-			builder.append(" url ["+metadata.get("url")+"]");
-			builder.append(" product ["+metadata.get("product")+"]");
-			builder.append(" product version ["+metadata.get("product-version")+"]");
-			builder.append(" driver ["+metadata.get("driver")+"]");
-			builder.append(" driver version ["+metadata.get("driver-version")+"]");
+		StringBuilder info = new StringBuilder();
+		if (metadata != null) {
+			info.append("user [").append(metadata.get("user")).append(CLOSE);
+			info.append("url [").append(metadata.get("url")).append(CLOSE);
+			info.append("product [").append(metadata.get("product")).append(CLOSE);
+			info.append("product version [").append(metadata.get("product-version")).append(CLOSE);
+			info.append("driver [").append(metadata.get("driver")).append(CLOSE);
+			info.append("driver version [").append(metadata.get("driver-version")).append(CLOSE);
 		}
-		builder.append(" datasource ["+obtainTargetDataSource().toString()+"]");
 
-		return builder.toString();
+		if (getTargetDataSource() instanceof OpenManagedDataSource) {
```

Comment:
Ik vind dit nog steeds geen handig plek, deze classe gaat over een datasource, en doet niets met pooling.
Je hebt aan de jms pooling properties gezeten, was het niet handiger geweest om een utililty te maken die uit een DataSources of (jms) ConnectionFactories de info haalt in plaats van dit (net als bij JMS) overall tussen te stoppen.

Reply:
Ik heb de package 'dbms' nu hernoemt naar 'datasource' aangezien dat beter matcht met de inhoud. Overige feedback hebben we besproken en kan eventueel later worden opgepakt, buiten deze PR.